dataland-frontend/src/components/forms/parts/fields/StringBadgeListFormField.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="form-field">
+    <div class="flex justify-content-between">
+      <UploadFormHeader :label="label" :description="description" />
+      <PrimeButton
+        :disabled="listOfElementsString === ''"
+        @click="addNewItems()"
+        :data-test="dataTestAddButton"
+        label="Add"
+        class="p-button-text"
+        icon="pi pi-plus"
+      ></PrimeButton>
+    </div>
+
+    <FormKit
+      :data-test="dataTestListInput"
+      type="text"
+      :ignore="true"
+      v-model="listOfElementsString"
+      placeholder="Add comma (,) for more than one value"
+    />
+    <FormKit
+      v-if="listOfElementsString.length > 0"
+      type="text"
+      v-model="listOfElementsString"
+      validation="length:0,0"
+      validation-visibility="live"
+      :validation-messages="{
+        length: 'Please add the entered value via pressing the add button or empty the field.',
+      }"
+      outer-class="hidden-input"
+    />
```

Comment:
Why do we need two `<FormKit>` elements to display only one actual input text field?
Is this a "workaround" to generate the "warning" that you need to click "+ADD" ?

Reply:
This is code from a former branch. I think I was involved and we had problems with the validation message being displayed at the right time. Therefore this solution. Feel free to try out a reduction to only one FormKit element.

Reply:
Ok no it's fine for me.   I was just curious.